.circleci/config.yml

```diff
@@ -121,6 +121,17 @@ jobs:
           command: inv -e lint-teamassignment
           name: run PR check for team assignment labels
 
+  milestone:
+    <<: *job_template
+    steps:
+      - restore_cache: *restore_source
+      - restore_cache: *restore_deps
+      - setup_remote_docker
```

Comment:
I don't think you need the `setup_remote_docker` directive, that attaches a disposable VM to execute docker commands.
While looking at the config.yml, seems the other lint steps also have this unnecessary directive

Reply:
:+1: